### ISSUE TITLE
puppet-lint should not warn "string containing only a variable" when it's used to create a hash

### ISSUE THREAD

**Issue by igalic** [rodjek/puppet-lint#261]
Given the code

``` puppet
  create_resources("gatekeeper2::install::${provider}", {
    "${title}" => {
      ensure => $ensure,
      target => $target,
      user   => $user,
      group  => $group,
    }
  })
```

puppet lint will warn, `manifests/install.pp - WARNING: string containing only a variable on line 48`. I find this rather unhelpful to automatically enforce style (in a git hook). Disabling it loses us a _lot_ of legitimate warnings.
